templates/api/json.adapter.h.tpl

```diff
@@ -6,13 +6,50 @@
 #endif
 #include "api.h"
 #include "apigear/utilities/qt_native.json.adapter.h"
+
```

Comment:
I don't like this empty line change mixed with other actual code changes.
There is no reason to change it here.
If you like to change the code style please use another commit to do so separately.